packages/core/src/agent/index.ts

```diff
@@ -46,7 +45,11 @@ import type {
 } from './types';
 
 export { MessageList };
+
 export * from './types';
+export * from './message-list/ai-sdk-5/index';
+export * from './v5-to-v4-stream';
+
```

Comment:
I'ts time to move agent to it's own file and make this a barrel file I guess, we shouldn't add these to this index file